src/libostree/libostree-released.sym

```diff
@@ -681,7 +681,7 @@ LIBOSTREE_2022.4 {
 global:
   ostree_fs_get_all_xattrs;
   ostree_fs_get_all_xattrs_at;
-} LIBOSTREE_2021.5;
+} LIBOSTREE_2022.2;
```

Comment:
To be extra clear, this section being changed here is the "prior version inheritance", *not* the version associated with the symbols, which is correctly `LIBOSTREE_2022.4`.